src/tree/tree.scss

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+$iui-child-node-spacing: calc(36px * (var(--level, 0)));
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+}
+
+@mixin iui-tree-node {
+  @include iui-focus;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding: $iui-xxs;
+  cursor: pointer;
+  box-sizing: border-box;
+  user-select: none;
+  margin-right: $iui-sm;
+  //min-height: $iui-component-height-small + (2 * $iui-xxs); //button height + padding
+  min-height: $iui-baseline * 3;
+
+  > .iui-icon {
+    @include iui-icons-default;
+    margin-right: $iui-s;
+    margin-left: $iui-sm + 1;
+  }
+
+  .iui-button + .iui-icon {
+    margin-left: $iui-xs + 1;
+  }
+
+  > .iui-button {
+    margin-right: $iui-s;
+    margin-left: $iui-s;
+
+    &:first-child {
+      margin-left: $iui-child-node-spacing;
+    }
+
+    > .iui-icon {
+      @media (prefers-reduced-motion: no-preference) {
+        transition: transform $iui-speed-fast ease-out;
+      }
+    }
+  }
+
+  > .iui-checkbox {
+    margin-left: $iui-s;
+    margin-right: $iui-child-node-spacing;
+  }
+
+  .iui-checkbox + .iui-node-label {
+    margin-left: $iui-s;
+  }
+
+  .iui-node-label {
+    min-width: 0;
+
+    &:only-child {
+      margin-left: $iui-child-node-spacing;
+    }
+  }
+
+  .iui-title,
+  .iui-caption {
+    white-space: nowrap;
```

Comment:
Can you add a very long string to HTML so we could really see that ellipsis is shown correctly?

Reply:
Added to With Caption

Reply:
Maybe we need to add padding on nodes or on all the tree itself because for me it doesn't look nice when text ends on the very beginning.
![image](https://user-images.githubusercontent.com/36186912/143187163-149164be-8e8a-4780-9eaf-1335266864ea.png)

Reply:
I added right margin to iui-node-label

Reply:
Better use padding for these kind of things.